projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid-filtering.spec.ts

```diff
@@ -0,0 +1,281 @@
+import { Component, ViewChild } from '@angular/core';
+import { async, TestBed, fakeAsync, tick } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { IgxTreeGridModule, IgxTreeGridComponent, IgxTreeGridRowComponent } from './index';
+import { IgxStringFilteringOperand, IgxNumberFilteringOperand, IgxDateFilteringOperand } from '../../../public_api';
+import { IgxTreeGridFilteringComponent } from '../../test-utils/tree-grid-components.spec';
+import { TreeGridFunctions } from '../../test-utils/tree-grid-functions.spec';
+import { configureTestSuite } from '../../test-utils/configure-suite';
+
+describe('IgxTreeGrid - Filtering actions', () => {
+    configureTestSuite();
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                IgxTreeGridFilteringComponent
+            ],
+            imports: [
+                BrowserAnimationsModule,
+                IgxTreeGridModule]
+        })
+        .compileComponents();
+    }));
+
+    it('should correctly filter a string column using the \'contains\' filtering conditions', () => {
```

Comment:
Separate the component creation in a beforeEach().

Reply:
Done